File: concrete/src/test/java/pcb/uwu/amount/composite/electromagnetism/HenriesTest.java
package pcb.uwu.amount.composite.electromagnetism;

import org.junit.Test;
import pcb.uwu.amount.quantity.Amperes;

import static java.math.MathContext.DECIMAL64;
import static org.junit.Assert.assertEquals;
import static pcb.uwu.core.Magnitude.MILLI;

public class HenriesTest {
	private static final Henries henries = new Henries(5);

	@Test
	public void multipliedByElectricCurrent() {
		assertEquals(new Webbers(35, MILLI),
				henries.multipliedBy(new Amperes(7, MILLI), DECIMAL64));
	}
}

File: implementation/src/test/java/pcb/uwu/amount/derived/electromagnetism/HenriesTest.kt
package pcb.uwu.amount.derived.electromagnetism

import pcb.uwu.amount.base.Amperes
import pcb.uwu.core.Magnitude.MILLI
import kotlin.test.Test
import kotlin.test.assertEquals

class HenriesTest
{
    @Test
    fun multipliedByElectricCurrent() =
        assertEquals(expected = Webbers(0.035),
                     actual = Henries(5) * Amperes(7, MILLI))

}